manifests/sriov-network-operator.package.yaml

```diff
@@ -1,4 +1,4 @@
 packageName: sriov-network-operator
 channels:
-- name: "4.4"
-  currentCSV: sriov-network-operator.v4.4.0
+- name: "4.5"
```

Comment:
Should this include the latest channel or all available channels?

Reply:
I think ART will create images for each release. Here we shall only put the latest one.